package.json

```diff
@@ -30,7 +30,8 @@
     "coverage-codecov": "cross-env NODE_ENV=test nyc report --reporter=text-lcov > coverage.lcov",
     "mocha": "cross-env NODE_ENV=test mocha test/ --recursive",
     "mocha-inspect": "cross-env NODE_ENV=test mocha --debug-brk --inspect test/ --recursive",
-    "migrations": "cross-env NODE_ENV=migration node migrations/",
+	"migrations": "node migrations/",
```

Comment:
shouldn't this use cross-env as well?

Reply:
I am not sure. In this case this script should be executed on the server and therefore should take the Env from the container into account.

Reply:
@FalkoBenthin ?

Reply:
`NODE_ENV=1 cross-env NODE_ENV=2 env | grep NODE_ENV ==> NODE_ENV=2` so yes, it would override the container env. Why do we need the `migration` environment?

Reply:
I do have a single migration database for testing. Thats just for local testing.